rust/extractor/src/translate/base.rs

```diff
@@ -160,7 +160,7 @@ impl<'a> Translator<'a> {
             self.path,
             start.line + 1,
             start.col + 1,
-            &message
+            &full_message
```

Comment:
I think more information is better in the logs, and I don't mind it being multiline. Maybe we can try this out, and revert it if we find it too spammy?

Reply:
In case of the semantic-less warning, I preferred the additional context of stating what will not work for the extraction of a given source file. Not much more information for us, but I think it gives a better context to a user other than hard-core rust analysis developers like us reading the logs :slightly_smiling_face: